lib/patch.js

```diff
@@ -201,80 +200,80 @@ function hasManyRemoting (relationName, relation, define) {
     }
   }, findHasManyRelationshipsFunc);
 
-  var createRelationshipFunc = function (cb) {
+  // var createRelationshipFunc = function (cb) {
```

Comment:
I'm a big fan of deleting rather than leaving commented code in.  We can always look at the commit history for reference.
Definitely a matter of preference though.